recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake
+
+
+class MysqlConnectorCppConan(ConanFile):
+    name = "mysql-connector-cpp"
+    version = "8.0.27"
+    license = "GPLv2"
+    author = "Oracle"
+    url = "https://dev.mysql.com/doc/connector-cpp/en/"
+    description = "MySQL Connector/C++ is a MySQL database connector for C++."
+    topics = ("mysql", "database", "connector", "c++")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "CMakeToolchain"
+    exports_sources = "CMakeLists.txt"
+    requires = "openssl/1.1.0l"
+    source_folder = "mysql-connector-cpp"
+    build_folder = "build"
+
+    def config_options(self):
+        pass
+        
+
+    def source(self):
+        self.run("git clone https://github.com/mysql/mysql-connector-cpp.git")
```

Comment:
please follow the pattern from https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L97-L98
so that the sources are retrieved from the tar.gz in `conandata.yml`